src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -669,6 +710,23 @@ public function testChgrpSymlink()
         $this->assertSame($group, $this->getFileGroup($link));
     }
 
+    public function testChgrpSymlinkById()
+    {
+        $this->markAsSkippedIfSymlinkIsMissing();
```

Comment:
symlink is not missing on appveyor, so this doesn't skip Windows.
use `if ('\\' === \DIRECTORY_SEPARATOR) {`

Reply:
I'm unable to understand the bug as is: I'll create a separate playground PR to debug it

Reply:
What about calling `markAsSkippedIfPosixIsMissing()` instead? I think `chgrp()` doesn't work otherwise.

Reply:
Yup, that's what I've done now: https://github.com/symfony/symfony/pull/35356/files#diff-f912c7214d9bb13479ead974deabe7d3R126